apps/web/screens/Article.tsx

```diff
@@ -263,13 +263,11 @@ const ArticleSidebar: FC<ArticleSidebarProps> = ({
       {article.subArticles.length > 0 && (
         <ArticleNavigation article={article} activeSlug={activeSlug} n={n} />
       )}
-      {article.relatedArticles.length > 0 && (
-        <RelatedContent
-          title={n('relatedMaterial')}
-          articles={article.relatedArticles}
-          otherContent={article.relatedContent}
-        />
-      )}
+      <RelatedContent
```

Comment:
Maybe we should check for `relatedContent.length` or `relatedArticle.length`? Just wondering what happens if there is neither?

Reply:
it is done inside `RelatedContent` so I  thought it would be redundant to have it here too :)